.ci/containers/downstream-builder/generate_downstream.sh

```diff
@@ -127,9 +127,22 @@ if [ "$REPO" == "terraform-validator" ] || [ "$REPO" == "tf-conversion" ]; then
 
     go mod tidy
 
+    # the following build can fail which results in a subsequent failure to push to tfv repository.
+    # due to the uncertainty of tpg being able to build we will ignore errors here
+    # as these files are not critical to operation of tfv and not worth blocking the GA pipeline
+    if [ "$COMMAND" == "downstream" ]; then
+      set +e
+    fi
+
+    mkdir -p ./testdata/generatedconvert/
```

Comment:
Why is this line necessary?

Reply:
removed, this was legacy from the previous revision